services/scabbard/libscabbard/src/service/factory.rs

```diff
@@ -34,7 +34,88 @@ const DEFAULT_STATE_DB_SIZE: usize = 1 << 30; // 1024 ** 3
 const DEFAULT_RECEIPT_DB_DIR: &str = "/var/lib/splinter";
 const DEFAULT_RECEIPT_DB_SIZE: usize = 1 << 30; // 1024 ** 3
 
-#[cfg(feature = "factory-builder")]
```

Comment:
This technically should be in the previous commit that removed factory-builder. (See other "this should match the struct comment".)

Reply:
Yep, missed that one